src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &--active {
+      background-image: url(/src/images/star-active.svg);
+    }
+
+    &--inactive {
+      background-image: url(/src/images/star.svg);
+    }
+  }
+}
```

Comment:
Your stars implementation should follow the stars task and set active stars based on a `stars--N` modifier [Reference](https://github.com/mate-academy/layout_product-cards/tree/master#:~:text=Rewrite%20stars%20block%20from%20the%20Stars%20task%20with%20SCSS%20and%20use%20it)